tests/Builder/Query/BitsAllClearOperatorTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Query;
+
+use MongoDB\BSON\Binary;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Query;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+/**
+ * Test $bitsAllClear query
+ */
+class BitsAllClearOperatorTest extends PipelineTestCase
+{
+    public function testBinDataBitmask(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::match(
+                a: Query::bitsAllClear(
+                    new Binary(base64_decode('IA==')),
+                ),
+            ),
+        );
+
+        $this->assertSamePipeline(Pipelines::BitsAllClearBinDataBitmask, $pipeline);
+    }
+
+    public function testBitPositionArray(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::match(
+                a: Query::bitsAllClear(1, 5),
+            ),
+        );
+
+        $this->assertSamePipeline(Pipelines::BitsAllClearBitPositionArray, $pipeline);
+    }
+
+    public function testIntegerBitmask(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::match(
+                a: Query::bitsAllClear(35),
```

Comment:
Related to my earlier question about why the YAML pipeline wraps `35` in an array, is `Query::bitsAllClear()` actually yielding an array-wrapped value? If so, that seems inconsistent with what the server would expect: https://www.mongodb.com/docs/manual/reference/operator/query/bitsAllClear/#mongodb-query-op.-bitsAllClear